static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -507,11 +508,12 @@ const spanOperationBreakdownRenderer = (field: string) => (
   const operationName = getSpanOperationName(field) ?? 'op';
 
   return (
-    <div style={{position: 'relative'}}>
+    <RowRectangleContainer>
       <RowRectangle
         spanBarHatch={false}
         style={{
           backgroundColor: pickBarColour(operationName),
+          top: 0,
           left: 0,
           width: toPercent(widthPercentage || 0),
```

Comment:
How is this overflowing if it's a % width? Shouldn't we constrain one of these inner containers instead of putting overflow: hidden everywhere?

Reply:
@k-fish I guess something like this:
<img width="1669" alt="Screen Shot 2021-05-03 at 5 39 48 PM" src="https://user-images.githubusercontent.com/139499/116937005-98ff1f00-ac36-11eb-8f2b-6950e4501c1e.png">

But could be addressed with https://github.com/getsentry/sentry/pull/25829

Reply:
@k-fish I guess something like this:

<img width="1653" alt="Screen Shot 2021-05-03 at 6 03 38 PM" src="https://user-images.githubusercontent.com/139499/116939197-f052be80-ac39-11eb-9524-129619f5a1b3.png">


But could be addressed with https://github.com/getsentry/sentry/pull/25829

Reply:
I'm fine with either, but if we go with this, I'd rather determine why the bar is exceeding it's container than add an overflow: hidden

Reply:
To be honest, I prefer https://github.com/getsentry/sentry/pull/25829